app/modules/cinema/endpoints_cinema.py

```diff
@@ -37,6 +44,27 @@
 hyperion_error_logger = logging.getLogger("hyperion.error")
 
 
+@module.router.get(
+    "/cinema/movie/{movie_id}",
+    response_model=schemas_cinema.Movie,
+)
+def get_movie(movie_id):
+    url = f"https://api.themoviedb.org/3/movie/{movie_id}?api_key={API_key}&language=fr-FR"
+    try:
+        r = get(url, timeout=5)
```

Comment:
You may prefer to use `import requests` and then `requests.get` instead of `get`

Reply:
You can pass query params directly to the `get` method
```suggestion
        response = get(
            url, 
            params: {
                "api_key": API_key,
                "language": "fr-FR",
            },
            timeout=5)
```

Reply:
You may also want to explicitly check if the API return an error indicating that the `movie_id` does not exist.
This would be an error caused be Titan that could be returned.